defaults-o2.sh

```diff
@@ -3,7 +3,6 @@ env:
   CMAKE_BUILD_TYPE: RELWITHDEBINFO
   CXXFLAGS: -fPIC -O2 -std=c++17
   CXXSTD: '17'
-  ENABLE_VMC: 'ON'
```

Comment:
Does it arm to leave it there for now? this will trigger a rebuild as well...

Reply:
ok, will leave that and only remove the corresponding line in `root.sh`

Reply:
no defaults are changed now